core/domain/config_services.py

```diff
@@ -35,7 +35,7 @@ def set_property(committer_id, name, value):
 
     config_property = config_domain.Registry.get_config_property(name)
     if config_property is None:
-        raise Exception('No config property with name %s found.' % name)
+        raise ValueError('No config property with name %s found.' % name)
```

Comment:
Please do not change the actual file.

Reply:
In that case i would use a much more general except catch, is that ok?